android/build.gradle

```diff
@@ -16,23 +16,27 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.1'
+        classpath('com.android.tools.build:gradle:3.4.1')
```

Comment:
I'd bump all these up - gradle 3.5.0, google-services 4.3.2, etc

Reply:
Update gradle and google services! perf-plugin and fabric were both updated in the initial commit.